persist/storer.go

```diff
@@ -95,6 +95,12 @@ func TmpPath(tempDir, integrationName string) string {
 // "nr-integrations" as a subfolder, Linux example: "/tmp/nr-integrations"
 func tmpIntegrationDir(tempDir string) string {
 	dir := tempDir
+
+	legacyCachePath := os.Getenv("NRIA_CACHE_PATH")
```

Comment:
perhaps moving it one layer up, inside TmpPath? Ideally I would do it outside TmpPath but i guess we want to automatically get this for all integrations using the sdk.

Reply:
make sense , i didn't noticed that other use